cmd/monaco/integrationtest/assert.go

```diff
@@ -318,14 +319,21 @@ func AssertBucket(t *testing.T, client client.BucketClient, env manifest.Environ
 	return expectedId
 }
 
-func getBucketWithRetry(ctx context.Context, client client.BucketClient, bucketName string, try, maxTries int) (buckets.Response, error) {
-	resp, err := client.Get(ctx, bucketName)
-	if try < maxTries && resp.StatusCode == http.StatusNotFound {
-		try++
+func getBucketWithRetry(ctx context.Context, client client.BucketClient, bucketName string, maxTries int) (buckets.Response, error) {
```

Comment:
Techlnically we could get rid of the response, because we don't need it anymore

Reply:
True 👍